### PR TITLE
Change certificate icon to a lock one

### LINKED ISSUE #8085: S/MIME: Alias Settings: Change certificate icon to lock icon
### Is your feature request related to a problem? Please describe.

The certificate icon is a bit confusing.

### Describe the solution you'd like

Use the lock icon instead.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #8085

### CODE REVIEW

**Comment by hamza221** [comment]
This won't be needed after #8175 gets merged

File: src/components/AliasForm.vue [hunk 1]
```diff
@@ -106,7 +106,7 @@ import moment from '@nextcloud/moment'
 import IconDelete from 'vue-material-design-icons/Delete'
 import IconRename from 'vue-material-design-icons/Pencil'
 import IconCheck from 'vue-material-design-icons/Check'
-import IconCertificate from 'vue-material-design-icons/Certificate'
+import IconCertificate from 'vue-material-design-icons/Lock'
```
Comment (ChristophWurst) [inline src/components/AliasForm.vue:109]:
```suggestion
import IconLock from 'vue-material-design-icons/Lock'
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Comment by GretaD** [comment]
this is ready to go if we decide to keep this PR

**Comment by hamza221** [comment]
#8175 is also done, so I think we can close this, @ChristophWurst  what do you think?

**Pull request closed without merging**